ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -404,6 +422,36 @@ - (MGLAnnotationImage *)mapView:(MGLMapView *)mapView imageForAnnotation:(id<MGL
     return annotationImage;
 }
 
+- (void)handleSingleTap:(UITapGestureRecognizer *)tap
+{
+    CLLocationCoordinate2D location = [_map convertPoint:[tap locationInView:_map] toCoordinateFromView:_map];
+    
+    NSDictionary *event = @{ @"target": self.reactTag,
+                             @"src": @{
+                                     @"latitude": @(location.latitude),
+                                     @"longitude": @(location.longitude),
```

Comment:
Maybe it would be worth sending along the screen coordinates (x, y) too, in case the developer wants to position some UI element over the map at the tap point. Or you could break out methods like `-convertPoint:toCoordinateFromView:` into separate methods that bridge directly between JS and MGLMapView. That could let developers do more with this functionality besides just handling taps.